orientation_readme.md

```diff
@@ -0,0 +1,39 @@
+# Orientation : Welcome to your first day!
+This document was made to help newcomers set up all the software that they will need to actively learn and participate in the JustAskDavidB Coaching Program.
+
+##  Downloading and Setting Up Software
+
+### [Jetbrain](https://account.jetbrains.com/login)
+We use Jetbrain products for web development.  This includes PHPStorm for PHP development, DataGrip for SQL and database interaction, and Pycharm for Python development. Downloading the trials is fine because a JetBrain license will be purchased for you as part of the JustAskDavidB Coaching Program. You will receive an email from JetBrains to complete your registration within the first 15-days. After logging into JetBrains, “Authorization Successful” should appear on screen.
+
+#### 1. [PHPStorm](https://www.jetbrains.com/phpstorm/download/#section=windows)
+Download the program for your OS and run through the installation. After installing Pycharm, open Pycharm and run through the setup wizard. Do not worry there are no wrong answers :)
+
+#### 2. [DataGrip](https://www.jetbrains.com/datagrip/download/#section=windows)
+Download the default “.exe” version. Run the datagrip-2023.2.1.exe file that starts the Installation Wizard. Follow all the steps suggested by the wizard. Pay special attention to the corresponding installation options. 
+
+#### 3. [PyCharm](https://www.jetbrains.com/pycharm/download)
+Download the default “.exe” version. Open the file and set up and install PyCharm. After setting up a trial on Jetbrain switch back to PyCharm and click the blue “Activate” button that has now appeared. Then click “Continue” at the bottom right-hand corner of the screen. PyCharm is now available to be used. 
```

Comment:
Add "and install PyCharm" to the end of the first sentence and delete the second sentence.
Change to "JetBrains" and add a comma
If they've already downloaded and started a trial with JetBrains at this point, there's no need to tell them to switch back to PyCharm. Instead you can say, "Click the blue "Activate" Button that should appear."